index.js

```diff
@@ -0,0 +1,24 @@
+import express from "express";
+import mongoose from "mongoose";
+import router from "./postUser/router.js";
+
+const port = process.env.PORT || 3000;
+const DB_URL = `mongodb+srv://admin:123qaz@cluster0.tihym.mongodb.net/myFirstDatabase?retryWrites=true&w=majority`;
```

Comment:
this value should be stored in proccess.env but its minor issue. you just need to know that is secure string